internal/services/network/application_gateway_resource.go

```diff
@@ -302,6 +303,75 @@ func resourceApplicationGateway() *pluginsdk.Resource {
 						},
 					},
 				},
+				Set: applicationGatewayBackendHttpSettingsHash,
+			},
+
+			"backend_settings": {
+				Type:         pluginsdk.TypeSet,
+				Optional:     true,
+				MinItems:     1,
+				RequiredWith: []string{"listener", "routing_rule"},
+				Elem: &pluginsdk.Resource{
```

Comment:
Does this conflicts with `backend_http_settings`?

Reply:
No, they can be configured together.